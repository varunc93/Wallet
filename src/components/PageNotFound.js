import React from 'react';
import {Link} from 'react-router-dom';

const PageNotFound = () => (
    <div>
        Error 404: Page Not Found! <Link to="/">Homepage</Link>
    </div>
);

export default PageNotFound;