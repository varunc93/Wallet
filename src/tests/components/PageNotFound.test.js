import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import PageNotFound from '../../components/PageNotFound'

test("Return DahBoardPage", () => {
    const wrapper = shallow(<PageNotFound />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});