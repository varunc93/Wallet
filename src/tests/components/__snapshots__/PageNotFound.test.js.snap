// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Return DahBoardPage 1`] = `
<div>
  Error 404: Page Not Found! 
  <Link
    replace={false}
    to="/"
  >
    Homepage
  </Link>
</div>
`;
